menu.rb

```diff
@@ -0,0 +1,148 @@
+# Brandi Wilson - RANDOM MENU GENERATOR (Dessert)
+
+# welcome & prompt
+question = "Would you like to see our menu (1) or create your own (2)?"
+puts "WELCOME TO FOOD WORLD".center(question.length)
+
+# list of words for the random menu
+food_adj = ["Gooey", "Crumbley", "Fun", "Questionable", "Mysterious", "Rainbow", "Mutant", "Magical", "Chilly", "Cheesy", "Spikey", "Cute", "Smooth", "Happy", "Hilarious", "Cat-loved", "Radioactive", "Watery", "Numeric", "Confused"]
+food_loc = ["Chinese", "Japanese", "British", "Scottish", "French", "Korean", "Indian", "Hawaiian", "American", "Canadian", "German", "Portugese", "Vietnamese", "Irish", "Italian", "African", "Australian", "Lihuanian", "Brazilian", "Mexican", "Egyptian"]
+food = ["Chicken", "Duck", "Baked Potato", "Eggplant", "Pie", "Ice Cream", "Fish", "Bread", "Mold", "Lettice", "Donut", "Coffee", "Tea", "Water", "Smoothie", "Burger", "Pizza", "Corn", "Rice", "Spaghetti", "Salad", "Apple"]
+
+# sorts the above arrays by length & assigns the shortest array length as the max number of available menu items
+max_lengths = [food_adj.length, food_loc.length, food.length].sort
+number_of_menu_items = max_lengths[0]
+
+while true   # requires if statement with "break" to break the loop
+
+  puts "#{question}"
+  user_input = gets.chomp.strip.to_i
+
+  # loop repeats until user enters a 1 or 2
+  until (user_input == 1) || (user_input == 2)
+    puts "Please enter 1 to view our menu or 2 to create your own."
+    user_input = gets.chomp.strip.to_i
+  end
+
+  # if the user wanted to see a pre-made menu
+  if user_input == 1
```

Comment:
Now that you know how to make methods - I'd be interested in seeing you possibly re-write this logic at some point to go through the two distinct paths using different methods